README.md

```diff
@@ -1,156 +1,202 @@
+| [**Bitcoin Secure Sequencer (bssd)**](https://github.com/hemilabs/heminetwork/blob/main/service/bss)  | Acts as a gateway to the Bitcoin Finality Governor (BFG), managing the consensus mechanisms of the Hemi Network. |
 
-* [PoP Miner (popmd)](service/popm): "mines" L2 Keystones into BTC blocks for proof-of-proof
-* [Bitcoin Finality Governor (bfgd)](service/bfg): Hemi's gateway to the BTC network.
-* [Bitcoin Secure Sequencer (bssd)](service/bss): Optimism's gateway to BFG, manages Hemi Network's consensus
+## 🌐 Binaries
 
-### License
+- [**bfgd (Bitcoin Finality Governor Daemon)**](cmd/bfgd): Manages connections and data transfers between the Hemi
+  Network and the Bitcoin blockchain, ensuring finality.
+- [**bssd (Bitcoin Secure Sequencer Daemon)**](cmd/bssd): Coordinates and sequences blockchain operations, serving as a
+  bridge to the Bitcoin Finality Governor.
+- [**extool**](cmd/extool): A utility tool for extracting and processing data from various file formats, tailored for
+  blockchain data analysis.
+- [**hemictl**](cmd/hemictl): The command-line interface for controlling and managing all Hemi Network services.
+- [**keygen**](cmd/keygen): Generates and manages cryptographic keys used within the Hemi network, ensuring secure
+  operations.
+- [**popmd (Proof-of-Proof Miner Daemon)**](cmd/popmd): Embeds L2 Keystones into Bitcoin blocks for proof-of-proof,
+  integral to the network's security.
+- [**tbcd (Tiny Bitcoin Daemon)**](cmd/tbcd): A minimal Bitcoin block downloader and indexer daemon.
 
-This project is licensed under the [MIT License](LICENSE).
+---
 
-## Getting Started
+# ⚡️ Getting Started
 
-### Building from Source
+## 📦 Downloading Release Binaries
 
-To build, you must have the following installed:
+Pre-built binaries are available on the [Releases Page](https://github.com/hemilabs/heminetwork/releases).
 
-* `git`
-* `make`
-* [Go v1.22+](https://go.dev/dl/)
+## 🏗 Building from Source
 
-First, clone the repository:
+### 🏁 Prerequisites
 
-```shell
-git clone https://github.com/hemilabs/heminetwork.git
-```
+- `git`
+- `make`
+- [Go v1.22+](https://go.dev/dl/)
 
-Then build:
+### Building with Makefile
 
-```shell
-cd heminetwork
-make deps
-make
-```
+1. Clone the `heminetwork` repository:
+   ```shell
+   git clone https://github.com/hemilabs/heminetwork.git
+   cd heminetwork
+   ```
 
-This will put built binaries in `<path-to-repo>/bin/`
+2. Setup and build binaries:
+   ```shell
+   make deps    # Download and install dependencies
+   make install # Build binaries
+   ```
 
-### Downloading Binaries from Releases
+Output binaries will be written to the `bin/` directory.
 
-You can find releases on the [Releases Page](https://github.com/hemilabs/heminetwork/releases)
+---
 
-### Running
+# 🛠 Running the Services
 
-To view options for any of the services, you may run the following
+For any service, you can view configuration options by running:
 
-```shell
+```bash
```

Comment:
Is there a reason to use the less generic `bash` over `shell` for syntax?  Not saying we have to change back but I have a slight preference for `shell` myself.

Reply:
That said, if we are being consistent and using `bash` everywhere we should stick with that.  Consistency over personal preferences :)

Reply:
`shell` should be used everywhere. I believe Jane changed it to use `bash` in most places for some reason. 
I have reverted, everything should be `shell` once again - thanks!